rlbot_gui/gui/csv/items.csv

```diff
@@ -2912,66 +2938,169 @@
 5313,Boost,Product_TA ProductsDB.Products.boost_toon_psplus,Blue Smoke
 5314,SupersonicTrail,Product_TA ProductsDB.Products.ss_ToonRocks_psplus,Blue Rocks
 5315,Skin,Product_TA ProductsDB.Products.skin_shark_lightning,Peregrine TT: Lightning
 5317,Wheels,Product_TA ProductsDB.Products.wheel_degree_psplus,Blue Notch
+5324,Antenna,Product_TA ProductsDB.Products.flag_mc_7lore,Lost Civilization
 5327,PlayerBanner,Product_TA ProductsDB.Products.playerbanner_huntress,Huntress
+5355,Antenna,Product_TA ProductsDB.Products.flag_mc_attlas,ATTLAS
+5356,Antenna,Product_TA ProductsDB.Products.flag_mc_FeedMe,Feed Me
+5357,Antenna,Product_TA ProductsDB.Products.flag_mc_Feint,Feint
+5358,Antenna,Product_TA ProductsDB.Products.flag_mc_FoxStevenson,Fox Stevenson
+5359,Antenna,Product_TA ProductsDB.Products.flag_mc_MYLK,MYLK
+5360,Antenna,Product_TA ProductsDB.Products.flag_mc_TokyoRose,Tokyo Rose
+5364,Body,Product_TA ProductsDB.Products.Body_Giftbox_Mystery_BlackMarket,Black Market Drop
+5365,Body,Product_TA ProductsDB.Products.Body_Giftbox_Mystery_Exotic,Exotic Drop
+5366,Body,Product_TA ProductsDB.Products.Body_Giftbox_Mystery_Import,Import Drop
+5367,Body,Product_TA ProductsDB.Products.Body_Giftbox_Mystery_Rare,Rare Drop
+5368,Body,Product_TA ProductsDB.Products.Body_Giftbox_Mystery_Uncommon,Uncommon Drop
+5369,Body,Product_TA ProductsDB.Products.Body_Giftbox_Mystery_VeryRare,Very Rare Drop
+5434,ShopItem,Product_TA ProductsDB.Products.ShopItem_Body_Endo,?INT?Products.ShopItem_Body_Endo.Label?
+5435,ShopItem,Product_TA ProductsDB.Products.ShopItem_Body_Flatbread,?INT?Products.ShopItem_Body_Flatbread.Label?
+5436,ShopItem,Product_TA ProductsDB.Products.ShopItem_Body_Mitty,?INT?Products.ShopItem_Body_Mitty.Label?
+5437,ShopItem,Product_TA ProductsDB.Products.ShopItem_Body_MuscleCar,?INT?Products.ShopItem_Body_MuscleCar.Label?
+5438,ShopItem,Product_TA ProductsDB.Products.ShopItem_Body_MuscleCar2,?INT?Products.ShopItem_Body_MuscleCar2.Label?
+5439,ShopItem,Product_TA ProductsDB.Products.ShopItem_Body_O2,?INT?Products.ShopItem_Body_O2.Label?
+5440,ShopItem,Product_TA ProductsDB.Products.ShopItem_Body_Pumpernickel,?INT?Products.ShopItem_Body_Pumpernickel.Label?
+5441,ShopItem,Product_TA ProductsDB.Products.ShopItem_Body_Rhino2,?INT?Products.ShopItem_Body_Rhino2.Label?
+5442,ShopItem,Product_TA ProductsDB.Products.ShopItem_Body_Sourdough,?INT?Products.ShopItem_Body_Sourdough.Label?
+5443,ShopItem,Product_TA ProductsDB.Products.ShopItem_Body_Takumi_II,?INT?Products.ShopItem_Body_Takumi_II.Label?
+5444,ShopItem,Product_TA ProductsDB.Products.ShopItem_Body_Torch2,?INT?Products.ShopItem_Body_Torch2.Label?
+5445,ShopItem,Product_TA ProductsDB.Products.ShopItem_Body_Twinzer,?INT?Products.ShopItem_Body_Twinzer.Label?
+5446,ShopItem,Product_TA ProductsDB.Products.ShopItem_Body_Wheatbread,?INT?Products.ShopItem_Body_Wheatbread.Label?
+5617,Skin,Product_TA ProductsDB.Products.skin_octane_elp,Octane: Pel�

```

Comment:
I noticed the �, can this be replaced with the right character, whatever that is?

Reply:
There are already multiple items with those. We would have to manually fix those every time we re-generate the file. Yes, you would only have to discard the changes, but sometimes the new items are spread all over the place so you'd have to manually search for them. But maybe there's a way to automate that?

Reply:
I noticed that the github csv preview shows the character correctly. Maybe we just have to change the file encoding or something, I'll investigate